src/frontend/components/DashboardVideoLiveControlPane/tab/DashboardVideoLiveTabAttendance/index.tsx

```diff
@@ -0,0 +1,125 @@
+import { Box, Heading, Spinner } from 'grommet';
+import React from 'react';
+import { defineMessages, FormattedMessage, useIntl } from 'react-intl';
+import { DashboardVideoSessionLiveTabAttendance } from 'components/DashboardVideoLiveControlPane/tab/DashboardVideoSessionLiveTabAttendance';
+import { ErrorMessage } from 'components/ErrorComponents';
+import { appData } from 'data/appData';
+import { useLiveAttendances } from 'data/queries';
+import { POLL_FOR_ATTENDANCES } from 'default/sideEffects';
+import { liveState, Video } from 'types/tracks';
+
+const messages = defineMessages({
+  loading: {
+    defaultMessage: 'Loading attendances...',
+    description:
+      'Accessible message for the spinner while loading the attendances.',
+    id: 'components.DashboardVideoLiveControlPane.tab.DashboardVideoLivettendance.loading',
+  },
+  noParticipants: {
+    defaultMessage: 'The live has no participant yet',
+    description: 'Message displayed if the live has no participant yet',
+    id: 'components.DashboardVideoLiveControlPane.tab.DashboardVideoLivettendance.noParticipants',
+  },
+});
+
+interface DashboardVideoLiveTabAttendanceProps {
+  video: Video;
+}
+
+export const DashboardVideoLiveTabAttendance = ({
+  video,
+}: DashboardVideoLiveTabAttendanceProps) => {
+  const intl = useIntl();
+  const interval =
```

Comment:
The naming is probably not the very accurate

Reply:
not sure what you meant I've renamed it to refetchInterval

Reply:
I ment, it seems like this is a boolean. But when I read `interval` or `refetchInterval` I rather expect a number.

Reply:
This is all the value accepted by the `refetchInterval` property use by `useQuery`: `refetchInterval: number | false | ((data: TData | undefined, query: Query) => number | false)`